bc_obps/registration/models.py

```diff
@@ -121,6 +121,7 @@ class Roles(models.TextChoices):
     role = models.CharField(
         max_length=1000,
         choices=Roles.choices,
+        blank=True,
```

Comment:
It is common to use `blank=True` to make a field not required on forms and the admin panel. This also applies to validation methods such as `full clean`. On the other hand, `null=True` allows a field to be saved as null in the database. However, for `Charfield/TextField`, it is unnecessary to specify `null=True` since it is best practice to save these fields as an empty string rather than null. 

For a more detailed explanation, you may refer to https://stackoverflow.com/questions/8609192/what-is-the-difference-between-null-true-and-blank-true-in-django#:~:text=Add%20a%20comment-,28%20Answers,-Sorted%20by%3A